prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -314,11 +299,32 @@ def prompt_processor() -> Any:
                                 "prompt_key": prompt_name,
                                 "doc_name": doc_name,
                             },
-                            LogLevel.ERROR,
+                            LogLevel.INFO,
                             RunLevel.RUN,
-                            msg,
+                            "Reading extracted file for context as chunk size is 0",
                         )
-                        raise APIError(message=msg)
+                except FileNotFoundError:
+                    msg = (
+                        "Extracted file not present. "
+                        "Please re-index the document and try again"
+                    )
+                    app.logger.error(
+                        "Extracted file for document is missing at %s",
+                        extract_file_path,
+                    )
+                    _publish_log(
+                        log_events_id,
+                        {
+                            "tool_id": tool_id,
+                            "prompt_key": prompt_name,
+                            "doc_name": doc_name,
+                        },
+                        LogLevel.ERROR,
+                        RunLevel.RUN,
+                        "Extracted file not present.",
+                    )
+                    return APIError(message=msg)
```

Comment:
@chandrasekharan-zipstack Currently this error is showing up as html. How do I throw the error in the right format?

Reply:
Let's huddle for this, didn't get you here @gaya3-zipstack